composeApp/src/commonMain/kotlin/org/ooni/probe/ui/run/RunViewModel.kt

```diff
@@ -278,3 +278,12 @@ class RunViewModel(
         data object DisableVpnInstructionsDismissed : Event
     }
 }
+
+private fun List<Descriptor>.runnableDescriptors(): List<Descriptor> {
+    return filter {
+        when (it.source) {
+            is Descriptor.Source.Default -> true
+            is Descriptor.Source.Installed -> !it.isExpired
+        }
+    }
+}
```

Comment:
The `Descriptor` model also has the `isExpired` field, so this `when` is not needed.